extra_geom/detectors.py

```diff
@@ -857,6 +857,43 @@ def from_simple_description(cls, pixel_size: float, slow_pixels: int, fast_pixel
 
         return geom
 
+    def inspect(self, axis_units='px', frontview=True, aspect='auto'):
+        """Plot the 2D layout of this detector geometry.
+
+        Returns a matplotlib Axes object.
+
+        Parameters
+        ----------
+
+        axis_units : str
+          Show the detector scale in pixels ('px') or metres ('m').
+        frontview : bool
+          If True (the default), x increases to the left, as if you were looking
+          along the beam. False gives a 'looking into the beam' view.
+        aspect : str, float
+          Set the aspect ratio of the plot, possible values:
+            * 'auto' (default): automatic; fill the position rectangle with data
+            * 'equal': same scaling from data to plot units for x and y
+            * a number: a figure will be stretched such that the height is num times the width.
+              aspect=1 is the same as 'equal'.
+        """
+        ax = super().inspect(axis_units=axis_units, frontview=frontview)
+
+        ax.set_title(f"{self.detector_type_name} geometry ({self.filename})")
+
+        scale = self._get_plot_scale_factor(axis_units)
+
+        # Label modules and tiles
+        for ch, module in enumerate(self.modules):
+            if self.n_tiles_per_module > 1:
```

Comment:
Do you want to add labels with just the module numbers for detectors with 1 tile per module?

Reply:
You're right, I overlooked this case! Thanks!
I changed to the following: 
* one module, one tile: no labels at all; 
* one module, `n+1` tiles: `T0 ... Tn`
* `n+1` modules, one tile per module: `M0`, `M1`, ... `Mn`
* many modules, many tiles: `M0T0 ... M0Tn`, ... `MkT0 ... MkTn`